src/App.tsx

```diff
@@ -69,6 +74,10 @@ const App = (): JSX.Element => {
         );
     }
 
+    if (!channel && !onLoginPage && !onResetPage) {
```

Comment:
Ultra nitpick, this might communicate the intent very slightly more clearly.

```suggestion
    if (!channel && !(onLoginPage || onResetPage)) {
```